### PR TITLE
Reposition study page action buttons 

### PR DESCRIPTION
**Pull request by alisman**
At Niki's request, move the study action buttons (circle icons) from the top right beside the gene query UI down beside the selection summary.  

image

### CODE REVIEW

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
```
Comment (alisman) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx]: this component was copied wholesale from RightPanel.tsx.  Code is the same.

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+@observer
+export default class ActionButtons extends React.Component<ActionButtonsProps, {}> {
+
+    @observable downloadingData = false;
+    @observable showDownloadErrorMessage = false;
+
+    @autobind
+    private handleDownload() {
```
Comment (adamabeshouse) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx]: initiateDownload? handle seems kind of vague

Comment (adamabeshouse) [reply src/pages/studyView/studyPageHeader/ActionButtons.tsx]: not important

File: src/pages/studyView/studyPageHeader/ActionButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        }
+    }
+
+    @computed
+    get virtualStudyButtonTooltip() {
+        //default value of userEmailAddress is anonymousUser. see my-index.ejs
+        return (
+            (_.isUndefined(this.props.user) ||
+                _.isEmpty(this.props.user) ||
+                _.isEqual(this.props.user.toLowerCase(), 'anonymoususer')
```
Comment (adamabeshouse) [inline src/pages/studyView/studyPageHeader/ActionButtons.tsx]: lets put "anonymousUser" in a const and reference it that way?

**Pull request merged**